summarization/stamp_page_picking/interesting_sequence_picker.py

```diff
@@ -62,7 +62,7 @@ def __init__(
         self._set_seed_stamp_page()
 
     def get_interesting_sequence_and_unused_pages(self):
-        for self.iteration_count in range(self.max_pages_allowed):
+        for self.iteration_count in range(self.max_pages_allowed-1):
```

Comment:
`self.iteration_count` need not be a class variable. We can simply use something like `i` here

Reply:
updated.